lib/src/widgets/pressable/pressable_widget.dart

```diff
@@ -72,7 +75,7 @@ class Pressable extends StatefulWidget {
   final VoidCallback? onLongPress;
   final FocusNode? focusNode;
   final bool autofocus;
-  final bool disabled;
+  final bool isDisabled;
```

Comment:
I think we can keep `disabled` instead of `isDisabled`

Reply:
Done 👍